content/blog/2018/08/2019-08-31-speaker-blog-delivery-automation.adoc

```diff
@@ -0,0 +1,82 @@
+---
+layout: post
+title: "Cultural Transformation via Delivery Automation"
+tags:
+- jenkins
+- jenkinsworld
+- jenkinsworld2018
+author: tanskann
+---
+
+NOTE: This is a guest blog by
+link:https://devopsworldjenkinsworld2018.sched.com/speaker/christian_weber.1y99b7id[Christian Weber] of
+link:https://github.com[GitHub] and
+link:https://devopsworldjenkinsworld2018.sched.com/speaker/tracy_ragan.70xae17[Tracy Ragan] of
+link:https://www.deployhub.com/[DeployHub].
+
+
+
+image::/images/conferences/devops-world-2018.jpg[DevOps World | Jenkins World 2018, float="left", link="https://www.cloudbees.com/devops-world"]
+
+Your Jenkins CI orchestration is all about workflow processing for your Dev process, but now you want to push continuous delivery.
+This of course means you need to have the pre-production and production teams on board with automation.
+Seems like an easy task, but you will soon find that cultural transformation in organizations is hard.
+Really hard.
+Whether you work in a large, small or somewhere in between organization, teams continuously struggle with changing cultural mindset to one of automation, inclusion and efficient workflow and design.

```

Comment:
"automation, inclusion and efficient workflow and design"

Unclear what is meant by inclusion and design.